src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/SqlAction.cs

```diff
@@ -668,4 +676,47 @@ LockOption lockOption
             LastModifiedAt: reader.GetDateTime(reader.GetOrdinal("LastModifiedAt"))
         );
     }
+
+    public async Task<string?> FindReferencingResourceNameByDocumentUuid(
+        DocumentUuid documentUuid,
+        int documentPartitionKey,
```

Comment:
It looks like we missed this on the `UpdateDocumentEdfiDoc` method as well.

Reply:
@simpat-adam 
I did not change the "UpdateDocumentEdfiDoc" method signature. Since, the DocumentUuid and PartitionKey values are coming from Db query.